packages/lucy/src/pocket.js

```diff
@@ -0,0 +1,97 @@
+export default function (part) {
+  const { options, Point, Path, points, paths, Snippet, snippets, complete, sa, paperless, macro } =
+    part.shorthand()
+
+
+    //pocket seams here 
+  let pocketLength = (800 * options.length)
+  let pocketWidth = (500 * options.width)
+  let pocketEdge = (options.edge)
+
+  points.topLeft = new Point(0, 0)
+  points.topRight = new Point(pocketWidth, 0)
+  points.bottomLeft = new Point(0, pocketLength)
+  points.bottomRight = new Point(pocketWidth, pocketLength)
+
+  //center point
+  points.center = new Point((pocketWidth / 2), 0)
+
+  //slit end 
+ points.middle = points.center.shift(270,(pocketLength*0.525)) 
+  
+ //top 
+
+ points.centerRight = points.center.shift(0, (pocketWidth * pocketEdge))
+ points.centerLeft = points.center.shift(180, (pocketWidth * pocketEdge))
+ 
+ //taper point 
+ points.taperRight = new Point(pocketWidth, (pocketLength * 0.4375))
+ points.taperLeft = new Point(0, (pocketLength * 0.4375))
+
+ //control points curve
+
+ points.leftCp1 = points.bottomLeft.shiftTowards(points.taperLeft, 50)
+ points.leftCp2 = points.bottomLeft.shiftTowards(points.bottomRight, 50)
+
+ points.rightCp1 = points.bottomRight.shiftTowards(points.taperRight, 50)
+ points.rightCp2 = points.bottomRight.shiftTowards(points.bottomLeft, 50)

```

Comment:
These control points would need to be either a shiftFractionTowards or the "50" would need to be a percantage of a value other wise this will not scale properly.

```suggestion
 points.leftCp1 = points.bottomLeft.shiftFractionTowards(points.taperLeft, 2 / 7)
 points.leftCp2 = points.bottomLeft.shiftFractionTowards(points.bottomRight, 0.2)

 points.rightCp1 = points.bottomRight.shiftFractionTowards(points.taperRight, 2 / 7)
 points.rightCp2 = points.bottomRight.shiftFractionTowards(points.bottomLeft, 0.2)

```
50 / 400 * 0.4375  = 50 / 175 = 2 / 7 for the vertical
50 / 250 = 0.2 for the horizontal.